clusterman/draining/queue.py

```diff
@@ -316,6 +329,9 @@ def process_drain_queue(
                     default=DEFAULT_DRAINING_TIME_THRESHOLD_SECONDS,
                 )
                 should_resend_to_queue = False
+                disable_eviction = (
+                    True if host_to_process.termination_reason == TerminationReason.SPOT_INTERRUPTION else False
+                )
```

Comment:
nit:
```suggestion
                disable_eviction = host_to_process.termination_reason == TerminationReason.SPOT_INTERRUPTION
```

we'll already get a `bool` from the comparison, so we can skip the `True if X else False` dance here :)

Reply:
OMG! how did i do this?! :D